test/jit/test_symbolic_shape_analysis.py

```diff
@@ -573,3 +573,14 @@ def too_many_inputs(x: List[int], y: List[int], z: Any, z2: Any):
             torch._C._jit_register_shape_compute_graph_for_node(node, too_many_inputs.graph)
 
         self.assertTrue("fewer arguments than schema" in str(error.exception))
+
+    def test_squeeze_dims(self):
+        def foo(input: torch.Tensor):
+            return input.squeeze([0, 2])
```

Comment:
can you also add a case for the issue that was failing before #98078 ?

Reply:
There was no case for which it was failing. I just wanted to modify this to have some convenience in Torch-MLIR.

Reply:
Didn't you link this error https://github.com/llvm/torch-mlir/actions/runs/4575347497 in #98078? What was the failure that was failing there which was fixed in #98078?

Reply:
That was because of the way sort op is handled in Torch-MLIR. Nothing related to shapes. In the modified implementation I just removed the usage of sort.